support/client/lib/vwf/view/jPlayer.js

```diff
@@ -0,0 +1,67 @@
+/// vwf/view/jPlayer.js is a sound/video driver
+/// 
+/// @module vwf/model/jPlayer
+/// @requires vwf/model
+
+define( [   
+        "module", 
+        "vwf/view", 
+        "jquery"
+    ], function( module, view, $ ) {
+    	return view.load( module, {
+            createdProperty: function (nodeID, propertyName, propertyValue) {
+                this.satProperty(nodeID, propertyName, propertyValue);
+             },
+
+            initializedProperty: function ( nodeID, propertyName, propertyValue ) { 
+                this.satProperty(nodeID, propertyName, propertyValue);
+            },
+
+            calledMethod: function( nodeID, methodName, methodParameters, methodValue ) {
+                var node = this.state.nodes[nodeID]; 
+
+                if( node ){
+                    var containerSelector = "#" + node.containerDivId;
+                    var playerSelector = "#" + node.playerDivId;
+
+                    switch ( methodName ){
+                        //We cannot safely call .show() or .hide() on the jPlayer div
+                        //See http://jplayer.org/latest/developer-guide/#jPlayer-disable-by-css
+                        case "show":
+                            var contX = node.containerSize[0];
+                            var contY = node.containerSize[1];
+                            var playerX = node.playerSize[0];
+                            var playerY = node.playerSize[1];
+                            
+                            $( containerSelector ).css('width', contX);
+                            $( containerSelector ).css('height', contY);
+                            $( playerSelector ).css('width', playerX);
+                            $( playerSelector ).css('height', playerY);
+                            break;
+                            
+                        case "hide":
+                            $( containerSelector ).css('width', 0);
+                            $( containerSelector ).css('height', 0);
+                            $( playerSelector ).css('width', 0);
+                            $( playerSelector ).css('height', 0);
+                            break;
+                    }
+                }
+            },
+
+    		satProperty: function ( nodeID, propertyName, propertyValue ) {
+                var node = this.state.nodes[nodeID];
+                if( node && propertyValue ){
+                    switch( propertyName ){
+                        case "containerDivSize":
+                            node.containerDivSize = propertyValue;
+                            break;
+                        case "playerDivSize":
+                            node.playerDivSize = propertyValue;
+                            break;
+                    }
+                }
+    		}
```

Comment:
That's really weird because it looks fine on my computer (see attached image). I've definitely pushed, and am on the branch/mg-staging+dev-jplayer-integration branch. 

![image](https://cloud.githubusercontent.com/assets/5407450/6279448/96c392fa-b86f-11e4-8b03-372c03101f1d.png)